gsitemap.php

```diff
@@ -533,8 +549,7 @@ protected function _getManufacturerLink(&$link_sitemap, $lang, &$index, &$i, $id
         foreach ($manufacturers_id as $manufacturer_id) {
             $manufacturer = new Manufacturer((int)$manufacturer_id['id_manufacturer'], $lang['id_lang']);
             $url = $link->getManufacturerLink($manufacturer, $manufacturer->link_rewrite, $lang['id_lang']);
-
-            $image_link = 'http'.(Configuration::get('PS_SSL_ENABLED') && Configuration::get('PS_SSL_ENABLED_EVERYWHERE') ? 's' : '').'://'.Tools::getMediaServer(_THEME_MANU_DIR_)._THEME_MANU_DIR_.((!file_exists(_PS_MANU_IMG_DIR_.'/'.(int)$manufacturer->id.'-'.ImageType::getFormattedName('medium').'.jpg')) ? $lang['iso_code'].'-default' : (int)$manufacturer->id).'-'.ImageType::getFormattedName('medium').'.jpg';
+            $image_link = $this->context->link->getBaseLink().Tools::getMediaServer(_THEME_MANU_DIR_)._THEME_MANU_DIR_.((!file_exists(_PS_MANU_IMG_DIR_.'/'.(int)$manufacturer->id.'-'.ImageType::getFormattedName('medium'))) ? $lang['iso_code'].'-default' : (int)$manufacturer->id).'-'.ImageType::getFormattedName('medium');
```

Comment:
I don't think this will work with `$this->context->link->getBaseLink()`. I'm not sure, but I'm under the impression you can get rid of it altogether.

Reply:
Please check this commit. I removed supplier and manufacturer because they are not used as pages I think?

Reply:
https://github.com/PrestaShop/gsitemap/pull/53/commits/91de6e87abad28c80692137bab532cf1e392ee13